File: hello.cpp
#include <ros/ros.h>                             // 引用 ros.h 檔
#include <ros/ros.h>                             // 引用 ros.h 檔

int main(int argc, char** argv){
    ros::init(argc, argv, "hello_cpp_node");     // 初始化hello_cpp_node
    ros::NodeHandle handler;                     // node 的 handler
    double frq;
    while (ros::ok()){                           // 在 ros 順利執行時
	handler.getParam("/print_freq", frq);
        ROS_INFO("Hello World! by cpp");                // 印出 Hello World
        ros::Duration(frq).sleep();                // 間隔 1 秒
    }
}

File: src/hello.py
#!/usr/bin/env python3
import rospy                             # import rospy 模組

rospy.init_node('hello_python_node')     # 初始化 hello_python_node

while not rospy.is_shutdown():           # 在 rospy 還沒結束前，執行下列指令:
    frq = rospy.get_param("/print_freq",1)
    rospy.loginfo('Hello World by py')         # 印出 Hello World
    rospy.sleep(frq)                       # 間隔 1 秒
